Cargo.toml

```diff
@@ -26,21 +27,24 @@ serde_derive   = "1.0"
 serde_cbor     = "0.11"
 atty           = "0.2"
 clap           = { version = "2.33", default-features = false }
```

Comment:
>  I'll gladly review such a PR.

OK, when I have time, I will try that. Will you be merging this PR (#496) soon? It would be easier for me to make my PRs on top of this one.

Reply:
> > I'll gladly review such a PR.
> 
> OK, when I have time, I will try that. Will you be merging this PR (#496) soon? It would be easier for me to make my PRs on top of this one.

Probably won't be merged any time soon. You can submit your PR directly against this branch: Checkout `version-0.4`, push a new branch, and create a PR (or draft PR) with the target being `version-0.4` from the drop-down menu.

Reply:
I had intended for command-line parsing and output and the like to start shifting over to `cargo-criterion` with the 0.4 release, so I think `clap` and `anes` and related crates should be behind the `cargo-bench-support` feature. That code would need to still be available in `criterion-rs` for the time being to give users time to migrate over to `cargo-criterion` (that's the purpose of the `cargo-bench-support` feature).